deps/rabbit/src/rabbit_amqqueue.erl

```diff
@@ -1896,7 +1726,7 @@ forget_node_for_queue(DeadNode, [H|T], Q) when ?is_amqqueue(Q) ->
         {false, _} -> forget_node_for_queue(DeadNode, T, Q);
         {true, rabbit_classic_queue} ->
             Q1 = amqqueue:set_pid(Q, rabbit_misc:node_to_fake_pid(H)),
-            ok = mnesia:write(rabbit_durable_queue, Q1, write);
+            ok = rabbit_db_queue:set_many([Q1]);
```

Comment:
That's because `set_many` it's the only set operation we have that just updates the durable queue. As `forget_node_for_queue` is to be removed when mnesia/transient entities are deleted, I thought that better stick to the API that we already have instead of add a `set_durable` that is going to be removed too. Sadly the queue API will need some changes once mnesia is gone, because we have to deal with durable and transient queues in a non-transparent way.